Gemfile

```diff
@@ -5,5 +5,9 @@ group :jekyll_plugins do
   gem 'jekyll-seo-tag'
 end
 
-gem 'wdm', '>= 0.1.0' if Gem.win_platform?
+group :development do
+  gem "jekyll"
+  gem "webrick"
```

Comment:
Jekyll has a known issue about webrick missing, might have been fixed in newer versions of jekyll